webui/react/src/contexts/AppContexts.tsx

```diff
@@ -9,8 +9,8 @@ import Users from 'contexts/Users';
 import usePolling from 'hooks/usePolling';
 import useRestApi from 'hooks/useRestApi';
 import {
-  getAgents, getCommands, getExperimentSummaries, getNotebooks, getShells,
-  getTensorboards, getUsers,
+  getAgents, getCommands, getExperimentSummaries, getNotebooks,
```

Comment:
nit: was it over the col limit?

Reply:
yeah, was hoping there is a linter that can auto fix this...